test/e2e/kuma_deploy_global_local_test.go

```diff
@@ -38,7 +35,7 @@ var _ = Describe("Test Local and Global", func() {
 		_ = clusters.DeleteKuma()
 	})
 
-	It("Should deploy Local and Global on 2 clusters", func() {
+	It("should deploy Local and Global on 2 clusters and sync dataplanes", func() {
```

Comment:
Remote and Global

Reply:
I didn't merge master yet, so I have a lot of "Local" usages in this PR